lte/gateway/c/core/oai/tasks/async_grpc_service/grpc_async_service_task.h

```diff
@@ -0,0 +1,248 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#pragma once
+
+#include <grpc++/grpc++.h>
+#include <grpcpp/impl/codegen/status.h>
+#include "feg/protos/s6a_proxy.grpc.pb.h"
+
+#include <memory>
+#include <atomic>
+
+namespace grpc {
+class ServerContext;
+}  // namespace grpc
+
+namespace magma {
+namespace feg {
+class CancelLocationRequest;
+class CancelLocationAnswer;
+class ResetRequest;
+class ResetAnswer;
+}  // namespace feg
+}  // namespace magma
+
+using grpc::ServerCompletionQueue;
+using grpc::ServerContext;
+using grpc::Status;
+
+namespace magma {
+
+void stop_async_s6a_grpc_server(void);
+using namespace feg;
+
+/**
+ * General async gRPC service. It runs a loop pulling requests off a server
+ * queue and then processing them. In general, the C++ implementation of async
+ * gRPC servers is quite confusing and requires a lot of groundwork. Hopefully
+ * this documentation can clear up some of the "magic".
+ *
+ * Every request is represented by a CallData, with one implementation per
+ * RPC call. The CallData has a state machine of the life cycle of the request.
+ * A request appears on the queue when it needs to be *processed* for the first
```

Comment:
why processed is between two asterisk?

Reply:
Not sure, copied from file, "SessionManagerServer.h"
@uri200 Any idea on this?

Reply:
I think that  is just for docstring formatting. I guess it will set that works as bold